src/pages/CreatePool/FeeTypeSelector.tsx

```diff
@@ -43,7 +43,7 @@ const ToggleWrapper = styled.button<{ background?: string }>`
   width: 100%;
 `
 
-export interface FeeTypeSelectorProps {
```

Comment:
do you know any tool that detect unused import?

Reply:
I used this https://marketplace.visualstudio.com/items?itemName=iulian-radu-at.find-unused-exports
There still many left in our codebase. Solving them take some effort so I suggest we do it gradually on each PR.
<img width="706" alt="image" src="https://user-images.githubusercontent.com/23650671/201454585-e1245a0c-ea4b-49a9-acd4-08f4d0bc85e5.png">